front/app/containers/IdeasShowPage/index.tsx

```diff
@@ -97,47 +96,41 @@ const IdeasShowPage = () => {
   const showCTABar =
     isIdeaInCurrentPhase && phase?.attributes.participation_method === 'voting';
 
-  const getPaddingTopIdeaContainer = () => {
-    const showDesktopCTABarAtTopOfPage = !isSmallerThanTablet && showCTABar;
-
-    if (isSmallerThanTablet) {
-      // This is the height of IdeaShowPageTopBar we show on tablet or smaller
-      return `${theme.mobileTopBarHeight}px`;
-    } else if (showDesktopCTABarAtTopOfPage) {
-      // This is the height of the CTA bar that might stick to the top of the page on desktop
-      return `${stylingConsts.menuHeight}px`;
-    }
-    return undefined;
-  };
-
   return (
     <>
       <IdeaMeta ideaId={idea.data.id} />
       <VotingContext
         projectId={project.data.id}
         phaseId={phaseContext || phase?.id}
       >
-        <Box background={colors.white} pt={getPaddingTopIdeaContainer()}>
+        <Box background={colors.white}>
           {isSmallerThanTablet ? (
             <StyledIdeaShowPageTopBar
               projectId={idea.data.relationships.project.data.id}
               ideaId={idea.data.id}
               phase={phase}
             />
           ) : (
-            <DesktopTopBar project={project.data} />
-          )}
-          <>
-            <Box mb="8px">
-              <main id="e2e-idea-show">
-                <StyledIdeasShow
-                  ideaId={idea.data.id}
-                  projectId={idea.data.relationships.project.data.id}
-                  compact={isSmallerThanTablet}
-                />
-              </main>
+            // 64px is the height of the CTA bar (see ParticipationCTAContent)
+            <Box mt={showCTABar ? '64px' : undefined}>
```

Comment:
Applying the margin-top here we're sure that we're on desktop when this renders (so there's no need for a check like `showDesktopCTABarAtTopOfPage`).